tools/cli/internal/changelog/changelog.go

```diff
@@ -77,7 +78,7 @@ type Path struct {
 	Tag            string     `json:"tag"`
 	StabilityLevel string     `json:"stabilityLevel,omitempty"`
 	ChangeType     string     `json:"changeType,omitempty"`
-	Changes        []Change   `json:"changes,omitempty"`
+	Changes        []*Change  `json:"changes,omitempty"`
```

Comment:
[q] just for my understanding, why are you using a pointer here?
what's the difference between `nil` and an empty slice?

Reply:
Because I am passing `changes` to different functions and it avoids copying the value on each of them and it should be more efficient during the append https://stackoverflow.com/questions/27622083/slices-of-structs-vs-slices-of-pointers-to-structs